samples/snippets/webhook.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2021, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START dialogflow_cx_webhook]
+
+import json
+
+
+def handleWebhook(request):
+
+    string_req = json.dumps(request)
+    req = json.loads(string_req)
```

Comment:
Is this roundtrip required? It looks like it's going from req(dict) -> string_req (str) -> req(dict).